api/base/filters.py

```diff
@@ -176,6 +189,11 @@ def parse_query_params(self, query_params):
                 # Special case date(time)s to allow for ambiguous date matches
                 if isinstance(field, self.DATE_FIELDS):
                     query[field_name].extend(self._parse_date_param(field, field_name, op, value))
+                elif not isinstance(value, int) and len(value.split(',')) > 1:
```

Comment:
Should this be checking `isinstance(field, IDField)` instead? If we only want to special case bulk `GET`s on IDs, that seems like a cleaner way of doing it.